apps/web/components/Navbar.js

```diff
@@ -39,7 +43,7 @@ const styles = StyleSheet.create({
     },
     shadowRadius: 3,
     paddingHorizontal: 10,
-    justifyContent: 'flex-start',
+    justifyContent: 'space-between',
```

Comment:
I am not entirely sure about this layout, but I guess we would need designs to tell us what we should do.

Reply:
It is how the web navbar is
![screenshot 2019-01-25 at 10 04 11](https://user-images.githubusercontent.com/7705041/51735789-98a37100-2088-11e9-8030-58f95ddb955b.png)